propcache/__init__.py

```diff
@@ -1,8 +1 @@
-from ._helpers import cached_property, under_cached_property
-
 __version__ = "0.1.0.dev0"
-
-__all__ = (
-    "cached_property",
-    "under_cached_property",
-)
```

Comment:
we could have an explicit `__all__ = ()` I suppose

Reply:
done, also added a comment in case anyone is wonder where they went